.env.example

```diff
@@ -1,24 +0,0 @@
-# Database credentials
-DB_HOST="127.0.0.1"
-DB_PORT="3306"
-DB_NAME="final_project_db"
-DB_USER="root"
-DB_PASSWORD="root"
-
-# Your API will run on this port
-API_PORT="5000"
-
-# The path the API will run from
-API_PATH="/api"
-
-# Secret key - make sure to replace with unique key 
-SECRET_KEY="[FILL RANDOM LONG AND VERY SECRET SECRET KEY FOR USE WITH AUTHENTICATION SERVICE HERE]"
-
-# Your Client will run on this port
-CLIENT_PORT="3000"
-
-# Enable or disable opening a new browser window every time you run webpack development server
-OPEN_BROWSER="true"
-
-# Turn this on for Knex to print SQL queries (only for debugging, don't use on production)
-# DEBUG=knex:query
```

Comment:
Woops. Please don't delete the .env.example file. This one serves as a template so it's easy to create a .env file which is the file where you will have all your local secrets and which is not checked into git. This one should be checked in so we can share the template.